platform/windows/display_server_windows.cpp

```diff
@@ -1632,8 +1631,7 @@ void DisplayServerWindows::window_set_current_screen(int p_screen, WindowID p_wi
 		Size2i wsize = window_get_size(p_window);
 		wpos += srect.position;
 
-		wpos.x = CLAMP(wpos.x, srect.position.x, srect.position.x + srect.size.width - wsize.width / 3);
-		wpos.y = CLAMP(wpos.y, srect.position.y, srect.position.y + srect.size.height - wsize.height / 3);
+		wpos = wpos.clamp(srect.position, srect.position + srect.size - wsize / 3);
```

Comment:
`platform/macos/display_server_macos.mm` have the same `wpos` code as well (and similar part in `update_mouse_wrap`, but it's using NSPoint, so probably should not be changed).

Reply:
Ah, good point, realized I didn't include `.mm` files, will add those in a bit